client/src/components/Profile.svelte

```diff
@@ -0,0 +1,298 @@
+                <button type="submit" class="btn-primary">Save Profile</button>
+                <button type="button" class="btn-primary" on:click={goToHomepage}>
+                    Go to Homepage
+                </button>
+            </div>
+        </form>
+    </div>
+</main>
+
+<style>
+    /* General Layout */
+    .profile-container {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100vh;
+        background-color: var(--background-color);
+        margin: 0;
+    }
+
+    .profile-card {
+        background: var(--card-background);
+        padding: 2rem;
+        border-radius: 10px;
+        box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+        width: 450px;
+        text-align: center;
+    }
+
+    h1 {
+        margin-bottom: 1.5rem;
+        color: var(--font-color);
+        font-size: 28px;
+    }
+
+    /* Input Styles */
+    .input-group {
+        margin-bottom: 1.5rem;
+        text-align: left;
+    }
+
+    .input-group label {
+        display: block;
+        margin-bottom: 0.5rem;
+        font-size: 14px;
+        color: var(--font-color);
+    }
+
+    .input-group input {
+        width: 100%;
+        padding: 0.75rem;
+        border: 1px solid var(--border-color);
+        border-radius: 5px;
+        font-size: 14px;
+    }
+
+    .readonly-input {
+        background-color: #e9ecef;
+        color: #6c757d;
+    }
+
+    /* Membership Form */
+    .membership-form {
+        display: flex;
+        gap: 0.5rem;
+    }
+
+    /* Membership Tags */
+    .membership-tags {
+        display: flex;
+        flex-wrap: wrap;
+        gap: 0.5rem;
+        margin-top: 1rem;
+
+    }
+
+    .tag-item {
+        display: flex;
+        align-items: center;
+        gap: 0.5rem;
+        background-color: violet; /* Teal for visibility */
+        color: white; /* White text for contrast */
+        padding: 0.5rem 1rem;
+        border-radius: 20px;
+        font-size: 14px;
+        font-weight: bold;
+    }
+
+    .tag-name {
+        flex: 1;
+        color: white; /* Ensures visibility */
+    }
+
+    .edit-input {
+        flex: 1;
+        padding: 0.5rem;
+        border-radius: 5px;
+        border: 1px solid var(--border-color);
+        font-size: 14px;
+    }
+
+    /* Buttons inside Membership Tags */
+    .remove-btn {
+        background: none;
+        border: none;
+        color: ghostwhite;
+        font-weight: bold;
+        cursor: pointer;
+    }
+
+    .remove-btn:hover {
+        color: #ff6f61; /* Red for delete hover */
+    }
+
+
+    /* Button Group */
+    .button-group {
+        display: flex;
+        justify-content: space-between;
+        margin-top: 1.5rem;
+    }
+
+    .btn-primary {
+        width: 48%; /* Make buttons consistent */
+        padding: 0.75rem 1.5rem;
+        background-color: coral; /* Warm orange */
+        color: whitesmoke;
+        border: none;
+        border-radius: 5px;
+        font-size: 16px;
```

Comment:
should this be 14 px to match the rest? (we have 28 and 14, I think is the only 16)